scripts/cubemx-importer.stanza

```diff
@@ -268,6 +287,24 @@ defn jtag-pins (pins:Tuple<Stm32Pin>) -> False|JtagOption:
 defn gpio-pins (pins:Tuple<Stm32Pin>) -> Tuple<GpioOption>:
   to-tuple(seq(GpioOption, pins-with-signal(pins, "GPIO")))
 
+defn adc-pin? (s:String) -> True|False :
+  val b = not empty?(regex-match("^ADC.*_IN.*", s))
+  ; println("%_ : %_" % [s, b])
+  b
```

Comment:
`not empty?(regex-match("^ADC.*_IN.*", s))`

Reply:
This file is used internally so the print statements are used for debugging. Hence additional return value.